Module-3-Javascript-Coding-Task/src/sum-all-numbers-nested.spec.js

```diff
@@ -0,0 +1,76 @@
+import { sumArray, sumNumbers } from './sum-all-numbers-nested'
+
+describe('sum-all-numbers-nested', () => {
+
+    describe('sumArray', () => {
+        it('should return 15 ', () => {
+            const array = [1, [2, [[[3]]]], [4, 5]]
+            const actual = sumArray(array)
+            const expected = 15
+
+            expect(actual).toEqual(expected)
+        })
+
+        it('should return 15', () => {
+            const array = [1, [2, [3], [4, "5", null, undefined, [NaN, Infinity], [true, false], { id: '1' }, 5]]]
+            const actual = sumArray(array)
+            const expected = 15
+
+            expect(actual).toEqual(expected)
+        })
+
+        it('should return empty string when undefined', () => {
```

Comment:
test title should reflect what the test does, it says it returns empty string, however it returns null

Reply:
corrected